HoloLensCommander/HoloLensCommander/MainWindowViewModel.cs

```diff
@@ -299,7 +441,16 @@ private void RegisterCommands()
             this.UninstallAllAppsCommand = new Command(
                 (parameter) =>
                 {
-                    this.UninstallAllApps();
+                    try
+                    {
+                        this.UninstallApp();
```

Comment:
Is this the right API?  Seems identical to the call made for this.UninstallAppCommand (as opposed to the allapps)